docs/user-guide.md

```diff
@@ -752,6 +753,21 @@ The Spark Operator provides limited support for resource quota enforcement using
 
 If you are running Spark applications in namespaces that are subject to resource quota constraints, consider enabling this feature to avoid driver resource starvation. Quota enforcement can be enabled with the command line arguments `-enable-resource-quota-enforcement=true`. It is recommended to also set `-webhook-fail-on-error=true`.
 
+## Running multiple instances of the operator within the same k8s cluster
+
+If you need to run multiple instances of the operator within the same k8s cluster. Therefore, you need to make sure that the running instances should not compete for the same custom resources or pods. You can achieve this:
+
+Either:
+* By specifying a different `namespace` flag for each instance of the operator.
+
+OR if you want your operator to watch specific resources that may exist on different namespaces:
```

Comment:
`on different namespaces ` -> `in different namespaces`.

Reply:
Done.